ocs_ci/utility/csr.py

```diff
@@ -60,3 +63,18 @@ def get_csr_resource():
     """
     logger.info("Retrieving CSR data")
     return ocp.OCP(kind='csr', namespace=constants.DEFAULT_NAMESPACE)
+
+
+def get_pending_csr():
+    """
+    Gets the pending CSRs
+
+    Returns:
+        list: list of pending CSRs
+
+    """
+    csr_conf = get_csr_resource()
+    return [
+        item['metadata']['name'] for item in csr_conf.data.get('items')
+        if not item.get('status')
```

Comment:
I was also thinking about handle this in my pr but wanted to have quick fix and I was not sure about possible statuses and so on. Are you sure there cannot be any other status reported? Like bad one? This case this condition will not work.

Reply:
I didn't see any other status. but pending CSRs are always empty dictionary